src/encoded/static/components/search.js

```diff
@@ -1625,9 +1626,9 @@ export const ResultBrowser = (props) => {
         );
     }
     return (
-        <div>
+        <React.Fragment>
```

Comment:
Similarly, seems like you could just delete the `<div>` as I could have done in encoded but missed this.